src/mmu/oam.rs

```diff
@@ -0,0 +1,32 @@
+use std::ops::{Index, IndexMut};
+
+const OAM_SIZE: usize = 0xa0; //  160
+const BANK_WIDTH: usize = 1 << 13;
+
+pub struct OAM {
+    data: Vec<u8>,
```

Comment:
Since the size should not be changed, a Box<[u8]> might be more appropriate.
If you make the change, add .into_boxed_slice() after the vec![] in the new() method

Reply:
8e157c8